tools/Customization/DevHome.Customization/ViewModels/Environment/DevDriveCardViewModel.cs

```diff
@@ -0,0 +1,57 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using CommunityToolkit.Mvvm.ComponentModel;
+using DevHome.Common.Environments.Models;
+using DevHome.Common.Models;
+using DevHome.Common.Services;
+using DevHome.SetupFlow.Utilities;
+
+using Dispatching = Microsoft.UI.Dispatching;
+
+namespace DevHome.Customization.ViewModels.Environments;
+
+/// <summary>
+/// View model for the card that represents a dev drive on the setup target page.
+/// </summary>
+public partial class DevDriveCardViewModel : ObservableObject
+{
+    private readonly Dispatching.DispatcherQueue _dispatcher;
+
+    private readonly DevHome.Common.Services.IDevDriveManager _devDriveManager;
+
+    public string DevDriveLabel { get; set; }
+
+    public ulong DevDriveSize { get; set; }
+
+    public ulong DevDriveFreeSize { get; set; }
+
+    public ulong DevDriveUsedSize { get; set; }
+
+    public double DevDriveFillPercentage { get; set; }
+
+    public string DevDriveUnitOfMeasure { get; set; }
+
+    [ObservableProperty]
+    private bool _isSelected;

```

Comment:
I find it odd that each item knows if it is selected or not.  What is this for?

Reply:
This is not needed, remnants of copied code. Removed it.